mintpy/save_kite.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+############################################################
+# Program is part of MintPy                                #
+# Copyright (c) 2013, Zhang Yunjun, Heresh Fattahi         #
+# Author: Marin Govorcin, April 2021                       #
+############################################################
+
+
+import sys
+import argparse
+import datetime as dt
+import numpy as np
+from mintpy.utils import readfile
+
+try:
+    from kite.scene import Scene, SceneConfig
+except ImportError:
+    raise ImportError('Can not import kite / pyrocko!')
+
+
+d2r = np.pi / 180.
+r2d = 180. / np.pi
+
+
+EXAMPLE = """example:
+  #for Velocity [use step to export coseismic displacement]:
+  save_kite.py geo/geo_velocity.h5 -g geo/geo_geometry.h5
+  save_kite.py geo/geo_velocity.h5 -g geo/geo_geometry.h5 -m geo/geo_maskTempCoh.h5
```

Comment:
In this case, the code will read linear velocity, which has the unit of m/year. Is this expected?

For other modeling code, e.g. GBIS, we usually convert the velocity to the corresponding displacement using DATE12, so that only the displacement in the unit of m is used.

Reply:
BEATv1.1 has a possibility to model interseismic velocities and tensile dislocations, so in those cases, this export would be needed. Example: https://pyrocko.org/beat/docs/current/examples/Rectangular_tensile.html

But for other event-type inversions, we need the displacements. Maybe would be good to add a WARNING/REMINDER message at the end of the code, if someone exports velocities accidentally.

Reply:
Thank you @mgovorcin for the explanation, these sound great, I will add more comments on the examples. We could set `--dset` option as required, to force the user to think about this before blindly running.